.travis.yml

```diff
@@ -36,37 +36,32 @@ matrix:
     - os: linux
       env: >
         TOOLCHAIN=clang-cxx17
-        PROJECT_DIR=examples/foo
+        PROJECT_DIR=examples/CppJwt
 
     - os: linux
       env: >
         TOOLCHAIN=gcc-7-cxx17
-        PROJECT_DIR=examples/foo
-
-    - os: linux
```

Comment:
> I have checked that no toolchains removed from CI configs, they are commented
out instead so other developers can enable them back easily and to simplify
merge conflict resolution. No

That's a requirement. Please comment them out, no need to remove:
- https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#excluding-tests

Reply:
> I have checked that for every commented out toolchain there is a link to the
broken CI build page or to the minimum compiler requirements documentation
so other developers can figure out what was the problem exactly. No. I have updated the package documentation to make it clear that this library is c++17 only

Add the link to documentation then.